app/javascript/components/admin/AdminTable.js

```diff
@@ -18,7 +18,7 @@ class AdminTable extends React.Component {
       table: {
         colData: [
           { label: 'Id', field: 'id', isSortable: true },
-          { label: 'Email', field: 'email', isSortable: true },
+          { label: 'Email', field: 'email', className: 'wrap', isSortable: true },
```

Comment:
Any reason to not apply to to all columns by default in the `CustomTable`?

Reply:
I tried that earlier. I decided not to apply that class name to all cells since it would at times break up words in a way that didn't make a lot of sense (see last cell in the screenshot). So I decided to only apply that class on a case by case basis. 
![image](https://user-images.githubusercontent.com/42656458/101825067-1e086100-3afb-11eb-8c85-10c9ff633e43.png)